rust/arrow/src/util/mod.rs

```diff
@@ -21,5 +21,6 @@ pub mod display;
 pub mod integration_util;
 #[cfg(feature = "prettyprint")]
 pub mod pretty;
+pub mod serialization;
```

Comment:
what do you think about making this `pub(crate)` or `pub(super)`? In general I think that we should avoid exposing public unless needed, so that it is easier to refactor without breaking public apis.

Reply:
Thanks, makes sense. I think it is indeed best to expose as little as internal code as possible :+1: